plugin/src/automation/lift/liftconfig.ml

```diff
@@ -300,16 +313,19 @@ let initialize_elim_types c env sigma =
  * Utility function: Map over the constructors of a type 
  *)
 let map_constrs f env typ sigma =
-  let ((i, i_index), u) = destInd typ in
-  let mutind_body = lookup_mind i env in
-  let ind_bodies = mutind_body.mind_packets in
-  let ind_body = ind_bodies.(i_index) in
-  map_state_array
-    (f env)
-    (Array.mapi
-       (fun c_index _ -> mkConstructU (((i, i_index), c_index + 1), u))
-       ind_body.mind_consnames)
-    sigma
+  if isInd typ then
```

Comment:
We can just assume it's inductive and remove this check, otherwise we're just deferring problems to later when we call this and we shouldn't.